doc/reference_coverage.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sect1 id="Coverage">
+  <sect1info>
+    <abstract>
+      <para>These functions operate on collections of polygonal geometry that form "implicit coverages" in which any edges shared by the polygons match exactly. These implicit coverages are fast to compute, and can be operated on with window functions, which retain the implicit topology inside the window partition while altering the edges.</para>
+    </abstract>
+  </sect1info>
+
+  <title>Coverages</title>
+
+  <refentry id="ST_CoverageInvalidLocations">
+    <refnamediv>
+      <refname>ST_CoverageInvalidLocations</refname>
+
+      <refpurpose>Window function that evaluates a set of polygons in the window partition to determine if they form a valid polygonal coverage.</refpurpose>
+    </refnamediv>
+
+    <refsynopsisdiv>
+      <funcsynopsis>
+        <funcprototype>
+          <funcdef>geometry <function>ST_CoverageInvalidLocations</function></funcdef>
+          <paramdef><type>geometry winset </type>
+            <parameter>geom</parameter></paramdef>
+          <paramdef><type>float8 </type>
+            <parameter>tolerance</parameter></paramdef>
+        </funcprototype>
+      </funcsynopsis>
+    </refsynopsisdiv>
+
+    <refsection>
+      <title>Description</title>
+
+      <para>Collects all the polygons in the window partition and then checks whether they form a "clean coverage", in which no polygons have any overlap, all vertices on shared edges are identical, and no gaps between polygons exist smaller than the tolerance value.</para>
+      <para>If polygons are found that are not clean relative to their neighbors, the problematic edges will be returned as MULTILINESTRING. For clean polygons the return value will be a NULL. Any non-polygonal or empty geometries on the input will be generate NULL returns.</para>
```

Comment:
```suggestion
      <para>If polygons are found that overlap or leave a gap between their neighbors, the problematic edges will be returned as MULTILINESTRING. For clean polygons the return value will be a NULL. Any non-polygonal or empty geometries on the input will be generate NULL returns.</para>
```

Reply:
well, it's actually a bit more than that - adjacent polygons must be edge-matched as well.

I like using the term "coverage-valid" to encapsulate all this conditions.

I'm intending to write a section earlier in the manual outlining the concept of a "simple features coverage".  It will include a detailed discussion of coverage validity (similar to the one [here](http://lin-ear-th-inking.blogspot.com/2022/07/polygonal-coverages-and-operations-in.html)).

Reply:
Maybe better to say "where polygons do not form a valid coverage, the problematic edges...". My issue was just with the term "clean" which means different things to different people.